### PR TITLE
Helm orion

### PR DESCRIPTION
**Pull request by gabcoyne**

### CODE REVIEW

**Comment by jamiedick** [comment]
@gabcoyne general question but is this something we should host in its own repo?

File: helm/orion-agent/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+description: Prefect orion application bundle
+type: application
+home: https://github.com/PrefectHQ
+version: 0.0.0
+appVersion: "latest"
+maintainers:
+  - name: gabcoyne
+    email: george@prefect.io
+sources:
+  - https://github.com/PrefectHQ/customer-success-recipes
```
Comment (jamiedick) [inline helm/orion-agent/Chart.yaml:12]: outdated repo name

File: helm/orion-agent/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        ports:
+        - containerPort: 4200
+      - name: agent
+        image: prefecthq/prefect:2.0b2-python3.8
+        command: ["prefect", "agent", "start", "kubernetes"]
+        imagePullPolicy: "IfNotPresent"
+        env:
+          - name: PREFECT_API_URL
+            value: http://orion:4200/api
+
```
Comment (jamiedick) [inline helm/orion-agent/manifest.yml:29]: remove whitespace

File: helm/orion-agent/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+1. Check Prefect agent connectionat
```
Comment (jamiedick) [inline helm/orion-agent/templates/NOTES.txt:1]: what is this?

File: helm/orion-server/values.yaml [hunk 4]
```diff
@@ -0,0 +1,141 @@
+
```
Comment (jamiedick) [inline helm/orion-server/values.yaml:1]: remvoe

File: helm/orion-server/values.yaml [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+# -- Orion APU/UI deployment
+api:
+  # Default values for Orion API.
+  enabled: true
+  service:
+    type: ClusterIP
+    port: 4200
+
+
```
Comment (jamiedick) [inline helm/orion-server/values.yaml:17]: remove

File: helm/orion-server/values.yaml [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+  image:
+    # -- Version tag, corresponds to tags at https://hub.docker.com/r/bitnami/postgresql/
+    tag: 14.3.0
+
+
+serviceAccount:
+   enabled: true
+
+
+
```
Comment (jamiedick) [inline helm/orion-server/values.yaml:141]: remove

File: helm/orion-server/values.yaml [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+    # -- Configures size of postgres PVC
+    size: 8Gi
+
+  # -- initial postgres user to create
+  initdbUser: postgres
+
+  image:
+    # -- Version tag, corresponds to tags at https://hub.docker.com/r/bitnami/postgresql/
+    tag: 14.3.0
+
```
Comment (jamiedick) [inline helm/orion-server/values.yaml:135]: remove

File: k8s/terraform/prefect-agent/README.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+A terraform module to deploy the prefect agent on Amazon EKS cluster.
+
+## Usage
+
+```hcl
+module "prefect_agent" {
+  source      = "path/to/prefect-agent"
+
+  prefect_api_secret_id = "xxxxxxxxxxxxxxxxx"
+}
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/README.md:16]: is this accurate?

File: k8s/terraform/prefect-agent/variables.tf [hunk 9]
```diff
@@ -0,0 +1,95 @@
+variable "api" {
+  default = "api.prefect.io"
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:2]: type and description

File: k8s/terraform/prefect-agent/variables.tf [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+variable "prefect_api_secret_id" {
+  description = "Secret ID for Prefect Cloud api key stored in AWS secrets manager"
+  type        = string
+}
+variable "prefect_secret_key" {
+  type        = string
+  description = "key of aws secrets manager secret for prefect api key"
+}
+variable "node_affinity" {
+  type = object({
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:13]: description

File: k8s/terraform/prefect-agent/variables.tf [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+    key      = string
+    operator = string
+    values   = tuple([string])
+  })
+  default = null
+}
+variable "logging_level" {
+  default = "INFO"
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:21]: description

File: k8s/terraform/prefect-agent/variables.tf [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+  type    = string
+
+}
+variable "prefect_labels" {
+  default = "[]"
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:26]: description

File: k8s/terraform/prefect-agent/variables.tf [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+  type    = string
+
+}
+variable "prefect_version" {
+  default = "latest"
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:31]: description

File: k8s/terraform/prefect-agent/variables.tf [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+}
+variable "namespace" {
+  default = "prefect"
+}
+variable "replicas" {
+  default = 1
+}
+variable "service_account_name" {
+  default = "prefect-agent"
+}
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:52]: type and descriptions plz

File: k8s/terraform/prefect-agent/variables.tf [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+variable "automount_service_account_token" {
+  type    = bool
+  default = true
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:55]: description

File: k8s/terraform/prefect-agent/variables.tf [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+}
+variable "request_cpu" {
+  default = "100m"
+  type    = string
+
+}
+variable "limit_cpu" {
+  default = "500m"
+  type    = string
+
```
Comment (jamiedick) [inline k8s/terraform/prefect-agent/variables.tf:94]: descrption and white space

File: prefect/flows/airbyte-orchestration/sync-airbyte-config.py [hunk 17]
```diff
@@ -1,12 +1,13 @@
...
+
 from prefect import Flow, task
 from prefect.engine.signals import FAIL
 from prefect.schedules.schedules import IntervalSchedule
 from prefect.tasks.airbyte.airbyte import AirbyteConfigurationExport
 from prefect.tasks.secrets import PrefectSecret
 
-import boto3
-
 schedule = IntervalSchedule(interval=timedelta(hours=6))
```
Comment (jamiedick) [inline prefect/flows/airbyte-orchestration/sync-airbyte-config.py:13]: can you move this to another PR? its not related to the helm chart

File: prefect/flows/dbt/rerun-models-from-failure/flow.py [hunk 18]
```diff
@@ -1,4 +1,4 @@
-from prefect import Flow, task, context
+from prefect import Flow, context, task
 from prefect.tasks.dbt import DbtShellTask
 from prefect.triggers import all_failed, any_successful
```
Comment (jamiedick) [inline prefect/flows/dbt/rerun-models-from-failure/flow.py:4]: same - please move to separate PR

**Review by jamiedick — DISMISSED** [review]

File: helm/orion-agent/manifest.yml [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
+roleRef:
+  kind: Role
+  name: flow-runner
+  apiGroup: rbac.authorization.k8s.io
+
+
```
[the remark this answers is not in the document]
Comment (gabcoyne) [reply helm/orion-agent/manifest.yml]: Removed file

File: README.md [hunk 20]
```diff
@@ -9,7 +9,7 @@
 
 <h2 align="center">Prefect Recipes</h2>
 
-  <p align="center">
+  <p align="center">i
```
Comment (zzstoatzz) [inline README.md:12]: was this intentional?

**Review by zzstoatzz — APPROVED** [review]
